### PR TITLE
SSOC22 Logo added

### LINKED ISSUE #58: Add SSOC logo
Add SSOC  logo in the readme after HSOC logo 

Adjust the height and width accordingly . 
The logo  is attached below 

IMG-20220820-WA0008

### PR DESCRIPTION
**Pull request by legendarykamal**
## Description

Please include a summary of the change and which issue is fixed.

Fixes #58

## Type of change

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this :

- [] Add a new project

- [] Modified or enhanced the existing projects

- [x] Improve the readme of any project

## Domain of Contribution

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this

- [] Project using only Solidity

- [] Project using only Javascript

- [] Project using only Python

- [] Project Test on the Remix IDE

- [x] Mention below if project uses other language and ide .

## Checklist ✓

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this

- [x] l have performed a self-review of my own code or work.

- [] I have commented on my code, particularly in hard-to-understand areas.

- [x] My changes generate no new warnings

- [] I'm HSOC'22 contributor

- [x] I'm SSOC'22 contributor

## Screenshots/Gif (Recommended)
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,9 @@
 
 <div align="center">
 
-<img src="./Images/Hsoc22Logo.jpg" width="20%" height="20%">
-  
+ <img src="./Images/Hsoc22Logo.jpg" width="20%" height="20%">
+ 
+ <img src="./Images/ssoc22.jpg" width="20%" height="40%">
```
Comment (Vikash-8090-Yadav) [inline README.md:36]: can' t the height of the both images  be same and also increase the space b/w them

Comment (legendarykamal) [reply README.md:36]: yep was thinking of it but the css wasnt getting applied to ssoc logo , let me look once more :eye:

Comment (Vikash-8090-Yadav) [reply README.md:36]: @legendarykamal  any updates

**Comment by legendarykamal** [comment]
@Vikash-8090-Yadav  I have updated with needed changes.
image

**Comment by Vikash-8090-Yadav** [comment]
Great @legendarykamal  ..

**Pull request merged**